regression/memsafety/built_from_end/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
Please add a comment at the end of this file to record any insights you've gained from your preliminary investigaton of the fallure.

Put at the end
```
--
--
comment...
```

Reply:
Will do, thanks, didn't know how to add comments to the test files